gui-doc/scribblings/gui/message-class.scrbl

```diff
@@ -10,12 +10,13 @@ A message control is a static line of text or a static bitmap. The
 @method[message% set-label]).
 
 
-@defconstructor[([label (or/c label-string? (is-a?/c bitmap%) 
+@defconstructor[([label (or/c label-string? (is-a?/c bitmap%)
                               (or/c 'app 'caution 'stop))]
-                 [parent (or/c (is-a?/c frame%) (is-a?/c dialog%) 
+                 [parent (or/c (is-a?/c frame%) (is-a?/c dialog%)
                                (is-a?/c panel%) (is-a?/c pane%))]
                  [style (listof (or/c 'deleted)) null]
                  [font (is-a?/c font%) normal-control-font]
+                 [color (is-a?/c color%) _unset]
```

Comment:
Would it make sense to make this `(or/c #f (is-a?/c color%))` and default to `#f`?